packages/playground-examples/copy/en/4-0/New Checks/Class Constructor Code Flow.ts

```diff
@@ -0,0 +1,19 @@
+//// { compiler: { ts: "3.9.0" } }
+//
+// In 4.0, we use code flow analysis to
```

Comment:
```suggestion
// In 4.0, we use control flow analysis to
```

(Unless you think 'code flow' is a better term than the internal term 'control flow'.)

Reply:
Nah, consistency is better here - I think the Swift team use code flow and I've been reading their docs lately